pkg/operator/noobaa/system_state.go

```diff
@@ -0,0 +1,698 @@
+			Spec: core_v1.ServiceSpec{
+				Type:     core_v1.ServiceTypeLoadBalancer,
+				Selector: mgmtSelectorLabels,
+				Ports: []core_v1.ServicePort{
+					{Port: 8080, Name: "mgmt"},
+					{Port: 8443, Name: "mgmt-https"},
+					{Port: 8444, Name: "md-https"},
+					{Port: 8445, Name: "bg-https"},
+					{Port: 8446, Name: "hosted-agents-https"},
+					{Port: 80, TargetPort: intstr.FromInt(6001), Name: "s3"},
+					{Port: 443, TargetPort: intstr.FromInt(6443), Name: "s3-https"},
+				},
+			},
+		},
+
+		ServiceS3: &core_v1.Service{
+			ObjectMeta: makeObjectMeta(ServiceS3Name),
+			Spec: core_v1.ServiceSpec{
+				Type:     core_v1.ServiceTypeLoadBalancer,
+				Selector: s3SelectorLabels,
+				Ports: []core_v1.ServicePort{
+					{Port: 80, TargetPort: intstr.FromInt(6001), Name: "s3"},
+					{Port: 443, TargetPort: intstr.FromInt(6443), Name: "s3-https"},
+				},
+			},
+		},
+
+		CoreApp: &apps_v1.StatefulSet{
+			ObjectMeta: makeObjectMeta(coreAppName),
+			Spec: apps_v1.StatefulSetSpec{
+				Replicas: makeInt32Ptr(1),
+				Selector: &meta_v1.LabelSelector{
+					MatchLabels: corePodSelectorLabels,
+				},
+				ServiceName: ServiceMgmtName,
+				Template: core_v1.PodTemplateSpec{
+					ObjectMeta: addLabels(makeObjectMeta(coreAppName), corePodLabels),
+					Spec: core_v1.PodSpec{
+						ServiceAccountName: serviceAccountName,
+						Containers: []core_v1.Container{{
+							Name:            coreAppName,
+							Image:           sys.Spec.Image,
+							ImagePullPolicy: core_v1.PullIfNotPresent,
+							ReadinessProbe: &core_v1.Probe{
+								// # https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes
+								// # ready when s3 port is open
+								TimeoutSeconds: 5,
+								Handler: core_v1.Handler{
+									TCPSocket: &core_v1.TCPSocketAction{
+										Port: intstr.FromInt(6001),
+									},
+								},
+							},
+							Resources: core_v1.ResourceRequirements{
+								// # https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+								Requests: core_v1.ResourceList{
+									core_v1.ResourceCPU:    resource.MustParse("500m"),
+									core_v1.ResourceMemory: resource.MustParse("1Gi"),
+								},
+								Limits: core_v1.ResourceList{
+									core_v1.ResourceCPU:    resource.MustParse("4"),
+									core_v1.ResourceMemory: resource.MustParse("8Gi"),
+								},
+							},
+							Ports: []core_v1.ContainerPort{
+								{ContainerPort: 80},
+								{ContainerPort: 443},
+								{ContainerPort: 8080},
+								{ContainerPort: 8443},
+								{ContainerPort: 8444},
+								{ContainerPort: 8445},
+								{ContainerPort: 8446},
+								{ContainerPort: 60100},
+							},
+							VolumeMounts: []core_v1.VolumeMount{
+								{MountPath: "/data", Name: datadirPVCName},
+								{MountPath: "/log", Name: logdirPVCName},
+							},
+							Env: []core_v1.EnvVar{
+								{Name: "CONTAINER_PLATFORM", Value: "KUBERNETES"},
+								// - name: CREATE_SYS_NAME
```

Comment:
are all these commented out pod spec env vars needed? they don't seem to be informing the code in any way currently

Reply:
You're right. Removed it.